test/bdd/features/did-orb.feature

```diff
@@ -121,10 +121,10 @@ Feature:
     Scenario: concurrent requests plus server shutdown tests
 
      # write batch of DIDs to multiple servers and check them
-     # TODO: When client sends request to "https://orb.domain1.com/sidetree/v1/operations,https://orb2.domain1.com/sidetree/v1/operations,https://orb.domain2.com/sidetree/v1/operations" to create 50 DID documents using 10 concurrent requests
+     When client sends request to "https://orb.domain1.com/sidetree/v1/operations,https://orb2.domain1.com/sidetree/v1/operations,https://orb.domain2.com/sidetree/v1/operations" to create 50 DID documents using 10 concurrent requests
```

Comment:
Should have removed the line bellow TODOs, we can do it in separate PR. Same for other lines.